MODULE.bazel

```diff
@@ -7,10 +7,15 @@ module(
 
 # Do not update to newer versions until you need a specific new feature.
 bazel_dep(name = "rules_license", version = "0.0.4")
+bazel_dep(name = "rules_python", version = "0.24.0")
 bazel_dep(name = "bazel_skylib", version = "1.2.0")
-bazel_dep(name = "rules_python", version = "0.10.2")
 
 # Only for development
 bazel_dep(name = "platforms", version = "0.0.5", dev_dependency = True)
 bazel_dep(name = "stardoc", version = "0.5.3", dev_dependency = True)
 bazel_dep(name = "rules_cc", version = "0.0.9", dev_dependency = True)
+
+# Find the system rpmbuild if one is available.
+find_rpm = use_extension("//toolchains/rpm:rpmbuild_configure.bzl", "find_system_rpmbuild_bzlmod")
```

Comment:
Should this be a dev dependency?

Reply:
Yes. Changed.  But perhaps this exposes a bzlmod bug.
I can't change use_repo to dev_dependency.
So perhaps this won't work for downstream dependencies. I can look into that in a second pass. Right now I only want to get a working CI run.

Reply:
>I can't change use_repo to dev_dependency.

`use_repo` on a `use_extension(dev_dependency=True)` behaves the way you'd expect it to.

Reply:
I'll take that to mean that it won't fail, but it's not what I expect.
My expectation is that  `name = use_extension(..., dev_dependency=True)` returns a callable if you are in dev mode and nothing in non-dev mode. That is, I expect it to blow up unless `use_repo` is also dev only, rather than return a callable that does nothing. 

But, that is irrelevant to this PR.